apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/charSequenceAssertions.kt

```diff
@@ -116,6 +164,15 @@ infix fun <T : CharSequence> Assert<T>.containsRegex(pattern: String): Assertion
  * @return An [Expect] for the current subject of the assertion.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
  */
+@Deprecated(
+    "Switch from Assert to Expect; will be removed with 1.0.0 -- see https://github.com/robstoll/atrium/releases/tag/v0.9.0#migration for migration hints and scripts.",
+    ReplaceWith(
+        "this.asExpect().contains(patterns).asAssert()",
```

Comment:
this need special treatment due to helper class (also re-apply the ReplaceWith to see if it works)

Reply:
Special treament like replacing the `patterns`? `ch.tutteli.atrium.api.infix.en_GB.patterns`

Reply:
See `contains(regexPatterns: RegexPatterns)` in api-infix, if you search for usages, then you find `expect contains regexPatterns(a, *aX)` which means it has to look like the following:
```suggestion
        "this.asExpect().contains(regexPatterns(patterns.pattern, *patterns.otherPatterns).asAssert()",
```